hermes/main.go

```diff
@@ -16,18 +16,16 @@ type app struct{}
 
 func (app) Manifest(context.Context) (*plugin.Manifest, error) {
 	m := &plugin.Manifest{Name: "hermes"}
-	m.ImportCobraCommand(cmd.NewRelayer(), "ignite")
+	m.ImportCobraCommand(cmd.NewHermes(), "ignite relayer")
```

Comment:
Won't this require changes on cli main again?

Reply:
Yes, I reverted because we need to launch a CLI version without the builtin relayer

Reply:
Yeah that makes sense. I think it's better like this as the app will stay compatible with v29 and v28. We should avoid breaking changes so it's great 🙌